csrankings-i.csv

```diff
@@ -357,6 +357,7 @@ Ivan I. Garibay,University of Central Florida,http://ivan.research.ucf.edu,rPuLf
 Ivan Kisel,Goethe University Frankfurt,https://www.fias.science/de/computerwissenschaften/gruppen/ivan-kisel,NOSCHOLARPAGE
 Ivan Laptev,Ecole Normale Superieure,https://www.di.ens.fr/~laptev,-9ifK0cAAAAJ
 Ivan Martinovic,University of Oxford,https://www.cs.ox.ac.uk/people/ivan.martinovic,CDK6C2cAAAAJ
+Ivan Oliveira Nunes,Rochester Institute of Technology,https://www.rit.edu/directory/ixdics-ivan-de-oliveira-nunes,2ITEX20AAAAJ

```

Comment:
Please add an entry for the DBLP alias "Ivan De Oliveira Nunes".

Reply:
This alias already exists in the dblp-aliases.csv file.

Reply:
All need to be in the csrankings CSVs.

Reply:
I see. Thanks. I've added that alias as well.